openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -84,7 +83,7 @@ def horiz_comp_to_geom_mean(self, ctx, imt):
 
     # List of the horizontal component definitions that can be converted into
     # geometric mean
-    tmp = ['AVERAGE_HORIZONTAL', 'GMRotI50', 'RANDOM_HORIZONTAL',
+    tmp = ['GEOMETRIC_MEAN', 'GMRotI50', 'RANDOM_HORIZONTAL',
            'GREATER_OF_TWO_HORIZONTAL', 'RotD50']
 
     # Apply the conversion
```

Comment:
lines 78 and 79 should be moved inside the if statement to prevent failures when the horcom is not in COEFF and COEFF_PGA_PGV